### DISCUSSION TITLE
Implementing a Anisotropic pair potential which applies a torque depending on the particles angular momenta.

### DISCUSSION THREAD

**Discussion by khofmann6** [glotzerlab/hoomd-blue#1743] [Support] [answered]
Hello, 

I'm currently trying to implement an Anisotropic pair potential. As a starting point, I used the files provided in 
hoomd-blue/example_plugins/pair_plugin/... and adapted them. This works perfectly. 

Now to my issue: I need the pair potential to calculate torques on the particles depended on the angular momentum of particle i and j. How can I access/read the current particle values like velocity or angular momentum of particle tag_i and tag_j inside "EvaluatorPairLJRot.h" (see attached .txt file) and use them for calculations.

Thank you!
[EvaluatorPairLJRot.txt](https://github.com/glotzerlab/hoomd-blue/files/14619471/EvaluatorPairLJRot.txt)

**Comment by joaander** [comment, marked as answer]
You cannot access the particle velocity or angular momentum. The AnisoPotentialPair infrastructure assumes that the energy, force, and torque on a pair of particles is a function of their relative positions and orientations only. You can derive a custom class from `ForceCompute` that relaxes these assumption.

**Reply by khofmann6** [reply]
Thank you, i will try this
